abci/client/errors.go

```diff
@@ -0,0 +1,25 @@
+package abcicli
+
+import (
+	"fmt"
+
+	"github.com/cometbft/cometbft/abci/types"
+)
+
+// ErrUnknownAbciTransport is returned when trying to create a client with an invalid transport option
+type ErrUnknownAbciTransport struct {
+	Transport string
+}
+
+func (e ErrUnknownAbciTransport) Error() string {
+	return fmt.Sprintf("Unknown abci transport: %s", e.Transport)
+}
+
+type ErrUnexpectedResponse struct {
+	Response types.Response
```

Comment:
Why is this of type `types.Response` and, in `abci/server/errors.go`, `ErrUnknownClientRequest.Req` is of type `*types.Request`?

Reply:
no reason, going to remove the pointer